README.md

```diff
@@ -52,36 +131,24 @@ systemctl status multipathd
 multipath -ll
 ```
 
-**Important:** When configuring Linux multipath devices, verify that the `find_multipaths` parameter in the `multipath.conf` file is disabled. In RHEL 7.x, remove the`find_multipaths yes` string from the `multipath.conf` file.
+##### 2.2 Configuring for Kubernetes users (RHEL)
+Create and set the relevant storage system parameters in the `/etc/multipath.conf` file. You can also use the default `multipath.conf` file, located in the `/usr/share/doc/device-mapper-multipath-*` directory.
 
-#### 3. Configure storage system connectivity
-3.1. Define the hostname of each Kubernetes node on the relevant storage systems with the valid WWPN(for Fibre Channel) or IQN(for iSCSI) of the node. 
-
-3.2. For Fibre Channel, configure the relevant zoning from the storage to the host.
-
-3.3. For iSCSI, perform the following steps:
-
-3.3.1. Make sure that the login to the iSCSI targets is permanent and remains available after a reboot of the worker node. To do this, verify that the node.startup in the /etc/iscsi/iscsid.conf file is set to automatic. If not, set it as required and then restart the iscsid service `$ service iscsid restart`.
-
-3.3.2. Discover and log into at least two iSCSI targets on the relevant storage systems. (NOTE: Without at least two ports, multipath device will not be created.)
+Verify that the `systemctl status multipathd` output indicates that the multipath status is active and error-free.
 
 ```bash
-$ iscsiadm -m discoverydb -t st -p ${STORAGE-SYSTEM-iSCSI-PORT-IP1}:3260 --discover
-$ iscsiadm -m node -p ${STORAGE-SYSTEM-iSCSI-PORT-IP1} --login
-
-$ iscsiadm -m discoverydb -t st -p ${STORAGE-SYSTEM-iSCSI-PORT-IP2}:3260 --discover
-$ iscsiadm -m node -p ${STORAGE-SYSTEM-iSCSI-PORT-IP2} --login
+yum install device-mapper-multipath
+modprobe dm-multipath
+systemctl enable multipathd
+systemctl start multipathd
+systemctl status multipathd
+multipath -ll
 ```
 
-3.3.3. Verify that the login was successful and display all targets that you logged into. The portal value must be the iSCSI target IP address.
-
-```bash
-$ iscsiadm -m session --rescan
-Rescanning session [sid: 1, target: {storage system IQN},
-portal: {STORAGE-SYSTEM-iSCSI-PORT-IP1},{port number}
-portal: {STORAGE-SYSTEM-iSCSI-PORT-IP2},{port number}
-```
+#### 3. Configure storage system connectivity
+3.1. Define the hostname of each Kubernetes node on the relevant storage systems with the valid WWPN(for Fibre Channel) or IQN(for iSCSI) of the node.
```

Comment:
1. delete "Kubernetes"
Define the hostname of each node on the ...
2. valid WWPN (for ...)
<needs a space between "WWPN" and "(for">
3. or IQN(for ...)
<needs a space between "IQN" and "(for">